<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseCreateDialog</class>
 <widget class="QDialog" name="CourseCreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建课堂</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/img_icon.png</normaloff>:/images/img_icon.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>课堂名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>教师登入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>学生登入密码</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_record_auto">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>自动录制</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_record_manual">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>手动录制</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_record_no">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>不录制</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_courseName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
   </property>
   <property name="placeholderText">
    <string>请输入课堂名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_teacherPwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
   </property>
   <property name="placeholderText">
    <string>请输入老师的登入密码 , 选填</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_studentPwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
   </property>
   <property name="placeholderText">
    <string>请输入学生的登入密码 , 选填</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>课程时长（分）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_minutes">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="placeholderText">
    <string>选填，默认60分</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
